tests/rptest/services/redpanda.py

```diff
@@ -1606,13 +1606,13 @@ def brokers(self, limit=None) -> str:
         return ",".join(self.brokers_list(limit))
 
     def brokers_list(self, limit=None) -> list[str]:
-        brokers = [self.broker_address(n) for n in self._started[:limit]]
+        brokers = [self.broker_address(n) for n in self.nodes[:limit]]
```

Comment:
i was under the impression that self._started would track the proper set of nodes if a test were doing something like stopping / starting some subset of self.nodes. under that assumption, calls like brokers_list etc... would seem to behave more intuitively because they only report brokers that have been started.

i dunno if that is accurate or correct, but this seems like it might change behavior in an unexpected way?

Reply:
Yeah, I wasn't sure whether the expectation for callers is actually to use the started nodes, or all nodes. For clients it seems like it might not matter, but looks like quite a few tests rely on the current behavior. I reverted the behavior for everything except the asserts.